app/forms/FormUtils.scala

```diff
@@ -42,6 +42,10 @@ trait FormUtils {
     fillForm(form, priorAmount, cya)
   }
 
+  def fillForm(form: Form[BigDecimal], cya: Option[BigDecimal]): Form[BigDecimal] = {
```

Comment:
Can you please add a test for this method?
It seams that fillExpensesFormFromPriorAndCYA is not used. Can you please double check and delete it if not needed?

Reply:
Will do